src/components/learning-guide-chart.coffee

```diff
@@ -48,10 +48,19 @@ module.exports = class LearningGuideChart
     @drawCircles(container, fields, points)
     @drawPlane(container, points)
     @drawXAxis(container, fields, points)
+    @drawTitle(container, guide)
+
+
+  drawTitle: (container, guide) ->
+    wrap = container.append('g')
+      .append("svg:text")
+      .attr("text-anchor","middle")
+      .attr("x", WIDTH/2)
+      .attr("y", 5)
+      .attr('class', 'main-title')
+      .text("Your Flight Path | "+guide.title+" | All Topics")
 
 
-  # Future improvement: Place clouds so they aren't
```

Comment:
Hah! You know, I haven't mentioned it yet since I figured we had plenty else to work on, but another neat thing would be to animate the plane flying along the path between the circles.  It's quite doable: http://bl.ocks.org/KoGor/8162640